generator/src/main/java/org/apache/kafka/message/checker/MetadataSchemaCheckerTool.java

```diff
@@ -81,8 +103,18 @@ public static void run(
                         ", and path2: " + path2);
                 break;
             }
+            case "verify-evolution-git": {
+                String fileCheckMetadata = namespace.getString("file");
+                String gitContent = GetDataFromGit(fileCheckMetadata);
+                EvolutionVerifier verifier = new EvolutionVerifier(
+                        CheckerUtils.readMessageSpecFromFile(Paths.get("").toAbsolutePath().getParent() + "/metadata/src/main/resources/common/metadata/" + fileCheckMetadata),
```

Comment:
in general I don't understand what the strategy is here for paths. if you are using a relative path, you need to document what directory we need to be in. how are we locating the git repository root?

Reply:
Ok yeah I added documentation in the help output stating that this tool is expected to be run from the `generator` directory. That was the assumption I was working under. Getting the git root repo is done in the `GetDataFromGit` function in `CheckerUtil.java`

Reply:
If we know where the git repo root is, why can't we just change to that directory rather than requiring a specific current working directory?

Reply:
I changed to code so it works anywhere within the kafka directory.